pytket/extensions/quantinuum/backends/leakage_gadget.py

```diff
@@ -116,7 +118,7 @@ def get_detection_circuit(circuit: Circuit, n_device_qubits: int) -> Circuit:
         detection_circuit.Measure(qb, bb)
         postselection_qubits.append(qb)
 
-    for q in end_circuit_measures:
+    for ps_b_index, q in enumerate(end_circuit_measures):
```

Comment:
Not sure how helpful this is, because there is still the second variable around?